test/test_freelancer_dispute_resolver.py

```
@@ -0,0 +1,213 @@
+        args=["The README is missing."],
+    )
+    # Try to submit again
+    try:
+        setup_unresolved["client"].send_transaction(
+            sender=setup_unresolved["client_account"],
+            contract_address=setup_unresolved["contract_address"],
+            function="raise_dispute",
+            args=["Trying to overwrite my evidence."],
+        )
+        pytest.fail("Expected an exception but none was raised.")
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)